DigitalLearningSolutions.Data/Services/DiagnosticAssessmentDataService.cs

```diff
@@ -27,11 +27,16 @@ public DiagnosticAssessmentDataService(IDbConnection connection, ILogger<Diagnos
         public DiagnosticAssessment? GetDiagnosticAssessment(int customisationId, int candidateId, int sectionId)
         {
             // NextTutorialID is the ID of the first tutorial in the section, according to Tutorials.OrderBy
-            // or null if the last in the section. Similar for NextSectionID, using SectionID and SectionNumber
-
-            // Find these by making a list of other tutorials in the course, to then find tutorials in this section,
-            // and other sections (because a section must contain at least one tutorial), using a similar approach to
-            // the one used in the TutorialContentService
+            // or null if the last in the section. Similar for NextSectionID, using SectionID and SectionNumber
+
+            // Find these by making a list of other tutorials in the course, to then find tutorials in this section,
+            // and other sections (because a section must contain at least one tutorial), using a similar approach to
+            // the one used in the TutorialContentService.
+
+            // Using this list of other tutorials in the course we can work out if there is another item in the
+            // section (if there is an viewable tutorial, or a post learning assessment, or consolidation material),
+            // and if there are other sections (valid tutorials with a different tutorial ID, or with assessments or
+            // consolidation material.
```

Comment:
This last bit in brackets seems to be more applicable to other items in the section rather than other sections, unless I'm misunderstanding it? 🙂

Reply:
Yeah I meant valid tutorials with a different section ID here, thanks for picking that up! Fixed in 7244c04 🙂